### DISCUSSION TITLE
Can I join your team?

### DISCUSSION THREAD

**Discussion by ghost** [ROM-builders/temporary#16959] [Q&A]
Assalamualaikum, @Apon77 sir 
I have successfully built ROM with Pull Request.
Derpfest (https://cirrus-ci.com/build/4514142195810304)

Cherish (https://cirrus-ci.com/build/6623634341494784)

VoltageOS (https://cirrus-ci.com/build/5198303642517504)

I also distributed the roms in our channel
https://t.me/rc3n10aupdates/2117

Please add me a verified user of ROM builders

If you think u can add me in brunch 

Repo: https://github.com/rinto02/RMX2020-CherishOS-tiramisu-rinto02

Repo: https://github.com/rinto02/RMX2020-DerpFest-AOSP-13-rinto02

I am unofficially maintaining these 2 ROMs
I have updates several builds in our telegram channel..

I appreciated for giving us such a free ROM builder organization. 

Please sir add me as push user
Thanks it's really helpful.

**Comment by Apon77** [comment]
please follow #learn in telegram group
